csm_test_utils/rds/rds_backup.py

```diff
@@ -0,0 +1,132 @@
+import datetime
+import logging
+import sys
+import time
+import requests
+import yaml
+import json
+
+from influx_line_protocol import Metric, MetricCollection
+from ocomone import setup_logger
+from requests import Response, HTTPError
+from datetime import datetime
+from ..common import Client, base_parser, sub_parsers
+
+
+BASE_URL = "https://rds.eu-de.otc.t-systems.com/v3"
```

Comment:
pass this as arg

Reply:
Done, pass endpoint as arg, add api version as const